dotsBkgItem.py

```diff
@@ -160,34 +161,36 @@ def addBkg(self, file, flopped=False): ## also used by saveBkg
     def saveBkg(self):
         if not self.bkg.isBackgroundSet:
             self.canvas.MsgBox("Set to Background inorder to save", 3)
+            return
```

Comment:
If you have a condition that is a "failure" you can `return` from the method immediately, rather than wrapping all the subsequent code in an `else`. It's a good pattern to follow to test your exceptional/error cases at the top,  and return immediately (eject! eject!) if anything is wrong.

This also keeps the *good* path through your method at the top level of indentation, rather than being nested.